src/frameworks/compose/discover/frameworkMatcher.ts

```diff
@@ -0,0 +1,95 @@
+import { FrameworkSpec, FileSystem } from "./types";
+
+/**
+ *
+ */
```

Comment:
Please add comment (or remove!).

Same for elsewhere!

Reply:
exported those methods for testing purpose.